[docs] Updates for Voyager 1.6.5 (#20897)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-voyager/reference/data-migration/import-data.md

| [--target-ssl-cert](../../yb-voyager-cli/#ssl-connectivity) <certificateName> | Path to a file containing the certificate which is part of the SSL `<cert,key>` pair. |
| [--target-ssl-key](../../yb-voyager-cli/#ssl-connectivity) <keyName> | Path to a file containing the key which is part of the SSL `<cert,key>` pair. |
| [--target-ssl-crl](../../yb-voyager-cli/#ssl-connectivity) <path> | Path to a file containing the SSL certificate revocation list (CRL).|

> | --target-ssl-mode | Specify the target YugabyteDB SSL mode from one of `disable`, `allow`, `prefer` (default), `require`, `verify-ca`, or `verify-full`. |

Reviewer: ```suggestion
| --target-ssl-mode | Specify the SSL mode for the target database as one of `disable`, `allow`, `prefer` (default), `require`, `verify-ca`, or `verify-full`. |
```

| Argument | Description/valid options |
| :------- | :------------------------ |

> | --batch-size <number> | Size of batches in the number of rows generated for ingestion when you import data to source-replica database. <br> Default: Oracle (10,000,000) and PostgreSQL(100,000) |

Reviewer: ```suggestion
| --batch-size <number> | Size of batches in the number of rows generated for ingestion when you import data to source-replica database. <br> Default: 10000000 (Oracle) or 100000 (PostgreSQL) |
```

| -h, --help | Command line help for import data to source-replica. |
| --oracle-home <path> | Path to set $ORACLE_HOME environment variable. `tnsnames.ora` is found in `$ORACLE_HOME/network/admin`. Oracle migrations only.|
| [--oracle-tns-alias](../../yb-voyager-cli/#ssl-connectivity) <alias> | TNS (Transparent Network Substrate) alias configured to establish a secure connection with the server. Oracle migrations only. |

> | --parallel-jobs <connectionCount> | The number of parallel batches issued to the source-replica database. <br> Default: Oracle(16) and PostgreSQL (If yb-voyager can determine the total number of cores N, then use N/2 as parallel jobs, else it falls back to 8) |

Reviewer: ```suggestion
| --parallel-jobs <connectionCount> | The number of parallel batches issued to the source-replica database. <br> Default: 16 (Oracle); or, if yb-voyager can determine the total number of cores N, then N/2, otherwise 8 (PostgreSQL) |
```

> | --start-clean | Starts a fresh import with data files present in the `data` directory.<br>If target YugabyteDB database has any non-empty tables, you get a prompt to continue the import without truncating those tables; if you go ahead without truncating, then yb-voyager starts ingesting the data present in the data files in non-upsert mode.<br> **Note** that for cases where a table doesn't have a primary key, duplicate data may be inserted. You can avoid duplication by excluding the table using `--exclude-table-list`, or by truncating those tables manually before using the `start-clean` flag. <br>Default: false<br> Accepted parameters: true, false, yes, no, 0, 1 |

Reviewer: ```suggestion
| --start-clean | Starts a fresh import with data files present in the `data` directory.<br>If the target YugabyteDB database has any non-empty tables, you are prompted to continue the import without truncating those tables; if you proceed without truncating, then yb-voyager starts ingesting the data present in the data files in non-upsert mode.<br> **Note** that for cases where a table doesn't have a primary key, duplicate data may be inserted. You can avoid duplication by excluding the table using `--exclude-table-list`, or by truncating those tables manually before using the `start-clean` flag. <br>Default: false<br> Accepted parameters: true, false, yes, no, 0, 1 |
```
